### PR TITLE
conan-frogarian: add page

### PR DESCRIPTION
**Pull request by bl-ue**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5070

### CODE REVIEW

File: pages/common/conan-frogarian.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# conan frogarian
+
+> Displays the conan frogarian.
+> More information: <https://github.com/conan-io/conan/pull/6760>.
```
Comment (sbrl) [inline pages/common/conan-frogarian.md:4]: Is there any documentation on the subcommand we can reference instead? Referencing the pull request seems rathe  unhelpful to me.

**Review by sbrl — COMMENTED** [review]
No idea what a `frogarian` is, but clearly it's useful somehow useful :P

File: pages/common/conan-frogarian.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/conan-frogarian.md:4]: I completely agree @sbrl, but it's practically a mystery!  There's not a single reference to it on the conan docs, and [the PR](https://github.com/conan-io/conan/pull/6760)'s description reads:
```
Changelog: omit
Docs: omit
```

>No idea what a frogarian is, but clearly it's useful somehow useful :P

It's an image of a frog with a sword.  The [conan docs 404 page](https://docs.conan.io/404) has it in more detail :)

It's arguable whether this page is even necessary, but I decided to go for it for completeness.

Comment (sbrl) [reply pages/common/conan-frogarian.md:4]: Oh, I see! If there isn't a specific docs page for this subcommand, let's link to the main docs or the main GitHub repository.

Comment (bl-ue) [reply pages/common/conan-frogarian.md:4]: Okay.  I'll do that.

**Review by navarroaxel — APPROVED** [review]

File: pages/common/conan-frogarian.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# conan frogarian
+
+> Displays the conan frogarian.
+> More information: <https://docs.conan.io/>.
+
+- Display the conan frogarian:
```
Comment (bl-ue) [inline pages/common/conan-frogarian.md:6]: @sbrl good place to ask: do we have a preference as to which verb to use: "Print", "Display", "Output", "Report", etc.?  I like "print" the best.  WDTY?

Comment (owenvoke) [reply pages/common/conan-frogarian.md:6]: I think I prefer `Display`, so probably fine to leave it as is. 👍🏻

**Review by owenvoke — APPROVED** [review]
Uh. 😂
Frogarian

**Comment by bl-ue** [comment]
Yep.  It's kind of silly 😆

**Pull request merged**